build_tools/cmake/iree_benchmark_suite.cmake

```diff
@@ -68,22 +68,20 @@ function(iree_benchmark_suite)
     PARSE_ARGV 0
     _RULE
     ""
-    "DRIVER;TARGET_BACKEND;TARGET_ARCHITECTURE"
+    "SUITE_NAME;DRIVER;TARGET_BACKEND;TARGET_ARCHITECTURE"
     "BENCHMARK_MODES;BENCHMARK_TOOL;MODULES;TRANSLATION_FLAGS;RUNTIME_FLAGS"
   )
 
   iree_validate_required_arguments(
     _RULE
-    "DRIVER;TARGET_BACKEND;TARGET_ARCHITECTURE"
+    "SUITE_NAME;DRIVER;TARGET_BACKEND;TARGET_ARCHITECTURE"
     "BENCHMARK_MODES;BENCHMARK_TOOL;MODULES"
   )
 
   iree_package_name(PACKAGE_NAME)
 
   # Add the benchmark suite target for a specific platform.
```

Comment:
nit: "Add the benchmark suite sub target." (or similar, since this is no longer required to be a "specific platform")